keras/layers/recurrent.py

```diff
@@ -456,3 +456,143 @@ def get_config(self):
                   "inner_activation": self.inner_activation.__name__}
         base_config = super(LSTM, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class BidirectionalRNN(MaskedLayer):
+    ''' Bidirectional wrapper for RNNs
+
+    # Arguments:
+        rnn: `Recurrent` object. 
+        merge_mode: Mode by which outputs of the forward and reverse RNNs will be combined. One of {sum, mul, concat, ave}
+
+    # TensorFlow warning
+        BidirectionalRNN only works with Theano for the time being.
+    
+    # Examples:
+    ```python
+    model = Sequential()
+    model.add(BidirectionalRNN(LSTM(10, input_shape=(10, 20))))
+    model.add(Activation('softmax'))
+    
+    model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
+
+    model.fit([X_train,], Y_train, batch_size=32, nb_epoch=20,
+              validation_data=([X_test], Y_test))
+    ```
+    '''
+    def __init__(self, rnn, merge_mode='concat', weights=None):
+        if K._BACKEND != 'theano':
+            raise Exception("BidirectionalRNN will only work with Theano.")
+        import copy
+        self.forward = rnn
+        self.reverse = copy.deepcopy(rnn)
+        self.merge_mode = merge_mode
+        if not weights:
+            weights = [None, None]
+        nw = len(weights)
+        self.forward.initial_weights = weights[:nw/2]
```

Comment:
When weights = None, weights is set to [None,None].

So nw=2,
nw/2=1

weights [:nw/2]=None,
weights [nw/2:]=None..

Just as you said.. its all None.